pkg/driver/driver.go

```diff
@@ -767,8 +770,20 @@ func (driver *Driver) ScrubEphemeralPods(podsDirPath string) error {
 			return filepath.SkipDir /* Skip processing current directory and continue processing other directories */
 		}
 
+		log.Tracef("Walking path [%s]", fileFullPath)
+		
+		if info.IsDir() && (strings.Count(fileFullPath, string(os.PathSeparator)) - baseDepth) == 2 && info.Name() != "volumes" {
```

Comment:
Could you please add a comment here why the difference needs to be 2 for others reading this code.?

Reply:
Done